spec/support/vcr_setup.rb

```diff
@@ -7,8 +7,15 @@
   config.hook_into :webmock
   config.ignore_localhost = true
   config.configure_rspec_metadata!
-  config.filter_sensitive_data('<HIDDEN_KEY>') { ENV.fetch('STRIPE_SECRET_TEST_API_KEY', nil) }
-  config.filter_sensitive_data('<HIDDEN_CUSTOMER>') { ENV.fetch('STRIPE_CUSTOMER', nil) }
-  config.filter_sensitive_data('<HIDDEN_ACCOUNT>') { ENV.fetch('STRIPE_ACCOUNT', nil) }
   config.ignore_hosts('localhost', '127.0.0.1', '0.0.0.0', 'api.knapsackpro.com')
+
+  # Filter sensitive environment variables
+  [
+    'STRIPE_SECRET_TEST_API_KEY',
+    'STRIPE_CUSTOMER',
+    'STRIPE_ACCOUNT',
+    'STRIPE_CLIENT_ID',
+  ].each do |env_var|
+    config.filter_sensitive_data("<HIDDEN-#{env_var}>") { ENV.fetch(env_var, nil) }
+  end
```

Comment:
Nice! I often like to simplify it a little bit:
```suggestion
  %w[
    STRIPE_SECRET_TEST_API_KEY
    STRIPE_CUSTOMER
    STRIPE_ACCOUNT
    STRIPE_CLIENT_ID
  ].each do |env_var|
    config.filter_sensitive_data("<HIDDEN-#{env_var}>") { ENV.fetch(env_var, nil) }
  end
```

Slightly cleaner and easier to add a new item.